### ISSUE TITLE
Suboptimal comma placement with new property

### ISSUE THREAD

**Issue by Timer** [kaelzhang/node-comment-json#21]
**Tell us about your environment**

* **Node Version:** v12.16.2
* **comment-json Version**: 3.0.2

**Please show your use case / code slices / code link that could reproduce the issue**

```js
const C = require("comment-json");
const o = C.parse(`{
  "test": true // hello
}`);
o.newProperty = true;
console.log(C.stringify(o, null, 2));

```

Actual output:
```jsonc
{
  "test": true // hello
  ,
  "newProperty": true
}
```

**What did you expect to happen?**

```jsonc
{
  "test": true, // hello
  "newProperty": true
}
```

**Are you willing to submit a pull request to fix this bug?**

With help, sure! Not sure how easy this is to fix.

**Comment by kaelzhang** [comment]
Yes, this is an known issue, and fixing this will introduce some breaking changes.

**Comment by Timer** [comment]
Please let me know if I can be of any assistance 😄

**Comment by kaelzhang** [comment]
Landed in `4.0.0`
